config/jobs/kubernetes-csi/csi-driver-nfs/csi-driver-nfs-config.yaml

```diff
@@ -23,56 +23,12 @@ presubmits:
         - runner.sh
         args:
         - ./.prow.sh
+        env:
+        - name: CSI_SNAPSHOTTER_VERSION
+          value: "v3.0.2"
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
         resources:
           requests:
             cpu: 2000m
-  - name: pull-csi-driver-nfs-e2e
```

Comment:
Is this change required for the snapshot CRD version changes?

Reply:
I had to fix a mistake from someone's prior PR, see the first commit in this PR: 1145b61d36404380468a184c2722363a8908a6fd